tn3270/telnet.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, terminal_type, is_tn3270e_enabled=True):
         self.iac_buffer = bytearray()
         self.records = []
 
-    def open(self, host, port, ssl_context=None, ssl_server_hostname=None):
+    def open(self, host, port, ssl_context=None, ssl_server_hostname=None, timeout=None):
```

Comment:
Just a minor suggestion would be to reorder the parameters (keep the SSL ones at the end) and make it clearer that the timeout is just for the TN3270 negotiation and not the TCP/IP connection or something else...

```suggestion
    def open(self, host, port, tn3270_negotiation_timeout=None, ssl_context=None, ssl_server_hostname=None):
```